spec/stringProcessing/getSentencesSpec.js

```diff
@@ -0,0 +1,9 @@
+var getSentences = require( "../../js/stringProcessing/getSentences.js" );
+describe("Get sentences from text", function(){
+	it("returns sentences", function () {
+		var sentence = "Hello. How are you? Bye";
+		expect( getSentences( sentence )[0] ).toBe( "hello");
+		expect( getSentences( sentence )[1] ).toBe( " how are you");
```

Comment:
It is because we split on periods. This means the second sentence has an extra space. 
Since we only need to count sentences, there is no need yet for sanitizing this.